.gitignore

```diff
@@ -1,17 +1,9 @@
-*.pyc
-*.pyo
-*.gourmet-plugin
```

Comment:
This is required: `gourmet-plugin` files are generated by `python setup.py build_i18n`, with the appropriate translations for the given locale.

It's not files that we would want to commit